temporalcli/commandsmd/commands.md

```diff
@@ -1519,17 +1519,15 @@ This command provides the following task queue statistics:
 - `ApproximateBacklogCount`: The approximate number of tasks backlogged in this
   task queue.  May count expired tasks but eventually converges to the right
   value.
-- `ApproximateBacklogAge`: How far "behind" the task queue is running. This is
-  the difference in age between the oldest and newest tasks in the backlog,
-  measured in seconds.
+- `ApproximateBacklogAge`: Approximate age of the oldest task in the backlog, based on its creation time, measured in seconds.
```

Comment:
thought quite a bit on this and I think I quite like this - I know @josh-berry, you mentioned you didn't like the presence of head of queue, creation time and oldest task coming together all at once. However, after also accounting @ShahabT 's comments, I believe this captures the essence of what we are trying to describe well without confusing the reader.

let me know what you think!

Reply:
👍 Sounds good. Please wrap to 80 columns though (otherwise the user-facing output will look wrong). :)